multi-int-store/src/test/java/com/connexta/multiintstore/common/DataRetrieverTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.common;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+import com.connexta.multiintstore.config.CallbackAcceptVersion;
+import java.util.Objects;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.RestTemplate;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("test")
+public class DataRetrieverTest {
+
+  @Autowired private DataRetriever dataRetriever;
+
+  @Autowired private RestTemplate restTemplate;
+
+  @Autowired private CallbackAcceptVersion callbackAcceptVersion;
+
+  @After
+  public void afterEach() {
+    Mockito.reset(restTemplate);
+  }
+
+  @Test
+  public void retrieverTest() throws Exception {
+    String url = "nice.local";
+    MediaType mediaType = MediaType.APPLICATION_JSON;
+    JSONObject jsonRetval = new JSONObject().put("Quality", "Test");
```

Comment:
✏️ Is this shorthand for `jsonRetrieval`?

Reply:
Yes, would you prefer that name instead?

Reply:
That's cool:+1: Just thought it might be a typo